### ISSUE TITLE
Specify additional executables from the API

### ISSUE THREAD

**Issue by micpp42** [gogcom/galaxy-integrations-python-api#145]
With GOG games, extra utilities, different launch options etc can be run from the launcher using an "Additional Executables" menu option. It's possible for the user to add extra options like this manually for any game. Is there any way of doing this through the API for platforms other than GOG?

**Comment by mbanczerowski** [comment]
No, currently it is impossible, but we're planning on such feature.
Probably this will be utilized as additional parameter for `launch_game` but there is no definite decision yet.
 
I'll tell more when we have some information to share. Stay tuned :)
